test/algebra-blank-to-var/dawg-syntax/bnodes/syntax-bnodes-02.json

```diff
@@ -4,9 +4,12 @@
     "type": "bgp",
     "patterns": [
       {
+        "type": "pattern",
+        "termType": "Quad",
+        "value": "",
```

Comment:
So apparently the `Util.objectify` method adds these values. Can we update that function to not have a value there? I don't think there's a reason to have it? Unless that randomly breaks something.